packages/fxa-auth-db-mysql/lib/db/schema/patch-097-098.sql

```diff
@@ -0,0 +1,194 @@
+--
+-- This migration introduces the following subscription-related data collections:
+--
+--  * Capabilities relevant to clients
+--  * Capabilities granted by products
+--  * Products enabled via subscription
+--  * Subscriptions enabled for accounts
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
```

Comment:
Little tip that we should probably have documented somewhere, if you generate these migration files using `scripts/migration.sh` it will also generate a little assertion that protects against missing migrations accidentally being skipped over in prod. It also saves the effort of manually adding this `SET NAMES` and the patch level stuff.